Dockerfile

```diff
@@ -6,6 +6,9 @@ WORKDIR /conjur
 
 COPY Gemfile /conjur/Gemfile
 ARG PUPPET_VERSION
-RUN env PUPPET_VERSION="$PUPPET_VERSION" bundle
+RUN env PUPPET_VERSION="$PUPPET_VERSION" bundle && cp Gemfile.lock /tmp
```

Comment:
Also:
- Can you also add here the comment about why we're copying the lockfile to /tmp?
- I think putting stuff in /tmp is [somewhat discouraged](https://github.com/moby/moby/pull/13587) since there's likelyhood that it may be over-ridden in the future by default so we may need a different dir.

Reply:
thanks made some changes